R/pit.R

```diff
@@ -0,0 +1,146 @@
+#' Probability integral transform
+#'
+#' The probability integral transform (PIT). LOO-PIT is achieved with weighted sample.
+#'
+#' @param x (draws) A [`draws_matrix`] object or one coercible to a
+#' `draws_matrix` object.
+
+#' @param y (observations) A 1D vector of observations. Each element of `y`
+#' corresponds to a column in `x`.
+
+#' @param loo_weights A [`draws_matrix`] object or one coercible to a
```

Comment:
highlight that by default these are expected to be log weights? What is more, do is log = TRUE a good default? Can we assume users to see this?

Reply:
I changed the default to FALSE, and used the same doc-string as `weight_draws` to highlight the default value.